app/models/repository.rb

```diff
@@ -1,3 +1,7 @@
 class Repository < KalibroClient::Entities::Processor::Repository
   include KalibroRecord
+
+  def self.latest(count=1)
+    all.sort { |one, another| another.id <=> one.id }.first(count)
```

Comment:
Maybe a third opinion would be nice @mezuro/core 

I just want to stress again: code readability is more important than performance, if you can address the perfomance issues by other techniques such as caching, just have a look through all the other projects. Any change should be subject of a retrospective rather tahn issue comments.

There will be no changes regarding this on this PR.